plutus-core/untyped-plutus-core/test/Transform/inlinePure3.uplc.golden

```diff
@@ -1 +1 @@
-(lam b_5 [ (lam x_8 (lam y_9 [ (con integer 1) x_8 ])) (con integer 1) ])
```

Comment:
Frankly I don't think these original golden files look right?

Reply:
I still don't know what to do/think about these 2 UPLC tests. What do you think I should do @effectfully